### PR TITLE
Add to release process documentation

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: docs/release-process.md [hunk 1]
```diff
@@ -160,6 +162,9 @@ hub pull-request
 
 ### Propose promotion of artifacts
 
+The `k8s-container-image-promoter` tool is `cip` from [kubernetes-sigs/k8s-container-image-promoter](https://github.com/kubernetes-sigs/k8s-container-image-promoter).
+The `gsutil` tool may be obtained from `pip3`.
```
Comment (johngmyers) [inline docs/release-process.md:166]: We should also add information about `promobot-files`, but I haven't used that yet.

File: docs/release-process.md [hunk 2]
```diff
@@ -172,14 +177,14 @@ git checkout -b kops_images_${VERSION}
...
-k8s-container-image-promoter --snapshot gcr.io/k8s-staging-kops --snapshot-tag ${VERSION} >> images.yaml
+k8s-container-image-promoter run --snapshot gcr.io/k8s-staging-kops --snapshot-tag ${VERSION} >> images.yaml
 ```
 
 You can dry-run the promotion with 
 
 ```
 cd ${GOPATH}/src/k8s.io/k8s.io
-k8s-container-image-promoter --thin-manifest-dir k8s.gcr.io
+k8s-container-image-promoter run --thin-manifest-dir k8s.gcr.io
```
Comment (johngmyers) [inline docs/release-process.md:187]: My attempts to dry-run the promotion all complained about my permissions, so this is not entirely accurate.

File: docs/release-process.md (hunk 1 above)
Comment (hakman) [reply docs/release-process.md:166]: Should we replace `k8s-container-image-promoter` with `cip`?

Comment (johngmyers) [reply docs/release-process.md:166]: Possibly. I'd like to hear what @justinsb thinks.

Comment (johngmyers) [reply docs/release-process.md:166]: Anyway, I have more changes pending. I'm trying to get the more straightforward ones in first.

Comment (hakman) [reply docs/release-process.md:166]: Fine by me, just meant that if the tool is `cip`, we may as well use that in commands below.

**Review by rifelpet — APPROVED** [review]

**Pull request merged**